src/Domain/ShoppingList.php

```diff
@@ -80,6 +87,14 @@ public function getItems(): ShoppingItemStack
         return $this->items;
     }
 
+    /**
+     * @return bool
+     */
+    public function isArchived(): bool
```

Comment:
Comment: good job on the getter! Now we just need a setter - `setArchived` - or an "action" method e.g. `markAsArchived()`.